external-crates/move/crates/move-analyzer/src/symbols.rs

```diff
@@ -276,26 +314,26 @@ pub struct FieldDef {
     pub start: Position,
 }
 
-/// Definition of a struct
 #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord)]
-pub struct StructDef {
-    pub name_start: Position,
-    pub field_defs: Vec<FieldDef>,
-    /// Does this struct have positional fields?
-    pub positional: bool,
-}
-
-impl StructDef {
-    pub fn name_start(&self) -> Position {
-        self.name_start
-    }
+pub enum MemberDefInfo {
```

Comment:
This change was done to be able to use one function (`add_member_use_def`) to add a `UseDef` for all relevant module members. Previously we had nearly identical functions to add them for functions and structs, and in this PR we'd need another one for enums